### PR TITLE
LPS-177843

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:stop

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
@alinedoleron We need also to create a page to display the My Apps Dashboard in addition to creating the empty state for the table. Could you please implement it? Thanks :)

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardTable.tsx [hunk 1]
```diff
@@ -19,54 +20,78 @@ export type AppProps = {
 };
 interface DashboardTableProps {
 	apps: AppProps[];
+	emptyStateMessage: {
+		description1: string,
+	description2: string,
```
Comment (naraandrade) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardTable.tsx:25]: @alinedoleron there's a format issue here, could you fix it please? Thanks :)

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardTable.tsx [hunk 2]
```diff
@@ -19,54 +20,78 @@ export type AppProps = {
...
-		<ClayTable borderless className="dashboard-table-container">
-			<ClayTable.Head>
-				<ClayTable.Cell headingCell>
-					<div className="dashboard-table-header-name">
-						<span className="dashboard-table-header-text">
-							Name
-						</span>
+export function DashboardTable({apps, emptyStateMessage}: DashboardTableProps) {
+	const {description1, description2, title} = emptyStateMessage;
+	if (!apps.length) {
```
Comment (naraandrade) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardTable.tsx:31]: @alinedoleron Please, add a blank line between the declaration of `description1, description2, title` and their usage.

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/DashBoardPage/DashboardPage.tsx [hunk 3]
```diff
@@ -14,7 +14,16 @@ import {initialDashboardNavigationItems} from './DashboardPageUtil';
 
 import './DashboardPage.scss';
 
-export function DashboardPage() {
+type DashBoardPageProps = {buttonMessage: string, messages: {
+	description: string;
+	emptyStateMessage: {
+		description1: string,
+		description2: string,
+		title: string},
```
Comment (naraandrade) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/DashBoardPage/DashboardPage.tsx:22]: @alinedoleron format issue here too.

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
Hi @naraandrade. I've made the requested changes. Thanks!

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
Resent here: https://github.com/naraandrade/liferay-portal/pull/167.

**Pull request closed without merging**